src/renderGraphiQL.js

```diff
@@ -19,9 +19,9 @@ export type GraphiQLOptions = {|
 |};
 
 // Ensures string values are safe to be used within a <script> tag.
-function safeSerialize(data) {
+function safeSerialize(data: mixed) {
```

Comment:
```suggestion
function safeSerialize(data: mixed): string {
```
Also, return values on those functions don't cause any issue with Flow we can add them.